.github/workflows/main.yml

```diff
@@ -51,8 +51,8 @@ jobs:
     -  name: Protobuf lint check
        run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
 
-    - name: Bazle lint check
-      run: ./buildifier --lint=warn --mode=check --warnings=all BUILD.bazel WORKSPACE oppia_android_test.bzl && echo "Bazle lint check completed successfully"
+    - name: Bazel lint check
+      run: ./buildifier --lint=warn --mode=check --warnings=all -r app data domain model testing utility BUILD.bazel WORKSPACE oppia_android_test.bzl && echo "Bazle lint check completed successfully"
```

Comment:
What about the top level workspace and build file? Can we run this on the whole project by default?

Reply:
We had included BUILD, workspace and oppia-android-test.bzl top level files. By default there are no options like being in the top directory and run the buildifier from there only.

Reply:
The more we add to this, the more I feel like we should be creating a script with all these statements. That way we can run them in pre-push hooks as well, what do you'll think?

Reply:
Specifically mentioning those files should be fine.

Also +1 to Vinita’s suggestion to use a script.

Reply:
ok, great to have a pre-push hook. I will check this out

Reply:
added script